pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_user_policy/guide.fr-fr.md

```diff
@@ -1,12 +1,12 @@
 ---
 title: "IAM pour VMware on OVHcloud - Comment associer un utilisateur à une politique IAM globale"
-excerpt: "Découvrez comment associer une identité OVHcloud à une politique IAM"
-updated: 2024-05-23
+excerpt: "Découvrez comment associer une identité à une politique IAM OVHcloud"
+updated: 2024-10-24
 ---
 
 > [!warning]
-> La fonctionnalité IAM pour VMware on OVHcloud est actuellement en phase bêta.
-> Ce guide peut donc être incomplet. Notre équipe reste disponible sur notre canal Discord dédié. N’hésitez pas à nous rejoindre et à nous contacter : <https://discord.gg/ovhcloud>. Posez des questions, donnez votre avis et interagissez directement avec l’équipe qui construit nos services Hosted Private Cloud.
+> La fonctionnalité IAM pour VMware on OVHcloud est actuellement en phase bêta. Ce guide peut donc être incomplet.
```

Comment:
Same remark as for the other guide. Is it still in beta? Then the information must be coherent for all related guides.